Framework/DataHandling/inc/MantidDataHandling/LoadNexusLogs.h

```diff
@@ -97,6 +97,11 @@ class DLLExport LoadNexusLogs : public API::Algorithm {
   /// Create a time series property
   Kernel::Property *createTimeSeries(::NeXus::File &file,
                                      const std::string &prop_name) const;
+
+  /// Returns true if the character is a control value or invalid
+  static bool isControlValue(const char &c, const std::string &propName,
```

Comment:
Is there any reason this needs to be a static on the class? Can it just be a simple anonymous function defined only in the `.cpp` file?